homeassistant/components/zoneminder.py

```diff
@@ -77,14 +77,14 @@ def login():
     if ZM['password']:
         login_post['password'] = ZM['password']
 
-    req = requests.post(ZM['url'] + '/index.php', data=login_post)
+    req = requests.post(ZM['url'] + '/index.php', data=login_post, verify=False)
```

Comment:
The certificate validation should be set to false by default. Add a configuration option to disable it.